docs/content/usage/widgets/process.md

```diff
@@ -147,6 +153,8 @@ Note all keywords are case-insensitive. To search for a process/command that col
 | `user`                          | `user=root`                           | Matches by user; supports regex                                                 |
 | `state`                         | `state=running`                       | Matches by state; supports regex                                                |
 | `()`                            | `(<COND 1> AND <COND 2>) OR <COND 3>` | Group together a condition                                                      |
+| `gmem`                          | `gmem > 1000 b`                       | Matches the gpu memory column in terms of bytes; supports comparison operators  |
```

Comment:
`gmem%` might also be good, though I can add this after.

Reply:
I can look into adding it into the mem% toggle too